kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/Burrow.java

```diff
@@ -116,6 +116,18 @@ private HttpUrl address(String... paths) {
         .collect(Collectors.toList());
   }
 
+  private Map<String, Object> getClusterDetail(String cluster) throws IOException {
```

Comment:
This doesn't feel like enough "meat" to warrant its own method and its only used in one place, so just stick it in the `getClusterBootstrapServers` method

Reply:
done